packages/amplication-plugin-api/prisma/schema.prisma

```diff
@@ -0,0 +1,39 @@
+datasource postgres {
+  provider = "postgresql"
+  url      = env("DB_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+  output   = "../../../node_modules/@amplication/prisma-clients/amplication-plugin-api"
+}
+
+model User {
+  id       String @id @default(cuid())
+  username String @unique
+  password String
+  roles    Json
+}
+
+model Plugin {
+  createdAt   DateTime @default(now())
+  description String?
+  github      String?
+  icon        String?
+  id          String   @id @default(cuid())
+  name        String?
+  npm         String?
+  pluginId    String?  @unique
+  updatedAt   DateTime @updatedAt
+  website     String?
+}
+
+model PluginVersion {
+  createdAt DateTime @default(now())
+  id        String   @id @default(cuid())
+  pluginId  String?
+  pluginIdVersion String @unique
```

Comment:
Are we using this because you could not create a unique composite key from Amplication?

Reply:
yes